src/routes/docs/products/messaging/fcm/+page.markdoc

```diff
@@ -57,9 +57,9 @@ Some additional configuration is required to enable push notifications in your m
 1. Add `google-services.json` at the root of your project.
 1. Add Google Services class path to your app-level Gradle dependencies block `"com.google.gms:google-services:4.4.0"`.
 1. Add Google Services plugin to your app-level Gradle in the plugins block as `"com.google.gms.google-services"`.
-1. Add notification handler service to `AndroidManifest.xml` inside the application tag, alongisde other activities. Find an example of this service in the [Send push notification](/docs/products/messaging/send-push-notifications#add-targets) journey.
+1. Your custom service extends to `FirebaseMessagingService`, so you don't need to add the notification handler service to AndroidManifest.xml. Find an example of this service in the [Send push notification](/docs/products/messaging/send-push-notifications#add-targets) journey.
 ```xml
-<service android:name="<YOUR_NOTIFICATION_HANDLER_SERVICE>" android:exported="false">
```

Comment:
These docs were correct. It's the docs mentioning an out of the box `io.appwrite.NotificationHandler` handler that's incorrect.